apps/jobs/views.py

```diff
@@ -272,6 +272,13 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
                 "message": "Please wait while your submission being evaluated!"
             }
             return Response(response_data, status=status.HTTP_200_OK)
+
+        try:
+            submission_meta_attributes = json.load(request.data.get('submission_meta_attributes'))
```

Comment:
How about if we use `if request.data.get('submission_meta_attributes'):` rather than a try and logging.info running for every submission